ivy_tests/test_ivy/test_functional/test_nn/test_losses.py

```diff
@@ -1,3 +1,4 @@
+# for review
 # global
 import numpy as np
 from hypothesis import given, strategies as st
```

Comment:
explicit test skipping based on `dtype` such as `if fw == "torch" and pred_dtype == "float16":` and `fw == "tensorflow" and true_dtype not in ["uint8", "int32", "int64"]:` in the tests should not be necessary. We should remove these and handle these using the `unsupported_dtype` attributes on the backend functions.